### DISCUSSION TITLE
i want to apply the theme orderedlist to a github-page:

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#33818] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello dear all, 

i want to apply the theme orderedlist to a github-page: 

https://github.com/orderedlist/minimal

how to achive this!?

**Comment by JamesMGreene** [comment, marked as answer]
You can find some documentation of how to install themes here:
https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll

Also, not positive if it's the same theme or not (the one you linked to looks like it hasn't been updated in years), but there is another more official Pages theme _also_ called Minimal that you can find here:
https://github.com/pages-themes/minimal
